__tests__/app.test.js

```diff
@@ -187,31 +187,171 @@ describe("GET/api/articles/:article_id/comments", () => {
         expect(msg).toBe("Not Found In The Database");
       });
   });
-  test('400: invalid data type requested by the client (string)', () => {
+  test("400: invalid data type requested by the client (string)", () => {
     const article_id = "banana";
     return request(app)
-    .get(`/api/articles/${article_id}/comments`)
-    .expect(400)
-    .then(({body: {msg}}) => {
-      expect(msg).toBe("Bad Request")
-    })
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
   });
-  test('400: invalid data type requested by the client (float)', () => {
+  test("400: invalid data type requested by the client (float)", () => {
     const article_id = 8.5;
     return request(app)
-    .get(`/api/articles/${article_id}/comments`)
-    .expect(400)
-    .then(({body: {msg}}) => {
-      expect(msg).toBe("Bad Request");
-    })
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
   });
-  test('400: invalid data type requested by the client (negative integer)', () => {
+  test("400: invalid data type requested by the client (negative integer)", () => {
     const article_id = -10;
     return request(app)
-    .get(`/api/articles/${article_id}/comments`)
-    .expect(400)
-    .then(({body: {msg}}) => {
-      expect(msg).toBe("Bad Request");
-    })
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("POST/api/articles/:article_id/comments", () => {
+  test("201: post a new comment on the given article", () => {
+    const article_id = 2;
+    const newComment = {
+      username: "butter_bridge",
+      body: "Being fit matters",
+    };
+    return request(app)
+      .post(`/api/articles/${article_id}/comments`)
```

Comment:
The API should return only the newly posted comment, which is an object, not an array. Please change this test to check for that object and make sure the new comment has the correct ID.